packages/webdriverio/src/commands/element/addValue.js

```diff
@@ -28,7 +28,7 @@
 import { transformToCharString } from '../../utils'
 
 export default function addValue (value) {
-    if (!this.isW3C) {
+    if (!this.isW3C || this.isMobileBrowser) {
```

Comment:
whats the difference between isMobileBrowser and isMobile?

Also, did you ensure that this fix works on both android AND safari ios?

Reply:
I have merely done any tests on this. I noticed this could be the reason why the failure happens. This PR is just a suggestion than an actual fix. I literally spent only an hr on this.

Reply:
`IsMobile` has an additional check to see if the capaibilities contain a `browserName`. Also I'm not set to test anything IOS yet.